opencart 2.2.0.0/upload/admin/controller/module/smaily_for_opencart.php

```diff
@@ -285,7 +285,7 @@ public function index() {
                 $data['sync_token'] = $this->config->get('smaily_for_opencart_sync_token');
             } else {
                 // Generate random token if not saved in db.
-                $data['sync_token'] = bin2hex(random_bytes(6));
+                $data['sync_token'] = bin2hex(openssl_random_pseudo_bytes(6));
```

Comment:
Is it safe to assume that PHP is compiled with OpenSSL? If not, then it might be safer to just use `hash('md5')` (or `hash('crc32')` if shorter length is a requirement).

Reply:
I think even easier would be just using `uniqid()` function. [Uniqid](https://www.php.net/manual/en/function.uniqid.php)

Reply:
Yes, that works also.

Reply:
Implemented uniqid() function in 740fa3bab2376f3dfb78de18855519b6f08a78a2